.vscode/settings.json

```diff
@@ -1,3 +1,9 @@
 {
-  "typescript.tsdk": "./node_modules/typescript/lib"
+  "typescript.tsdk": "./node_modules/typescript/lib",
+  "workbench.colorCustomizations": {
+    "titleBar.activeBackground": "#38ceb5",
+    "titleBar.inactiveBackground": "#38ceb599",
+    "titleBar.activeForeground": "#15202b",
+    "titleBar.inactiveForeground": "#15202b99"
```

Comment:
Not entirely sure if this is necessary, but doesn't hurt anything ;)

Reply:
Yeah, didn't mean to commit that 🤦‍♂